thesis-api-v2/src/main/java/pl/edu/agh/quizzesthesis/business/service/QuestionService.java

```diff
@@ -17,24 +21,46 @@ public class QuestionService {
     private final DefinitionService definitionService;
     private final DefinitionProcessingService definitionProcessingService;
     private final WrongAnswerService wrongAnswerService;
+    private final UnsplashApiService unsplashApiService;
     private final TermMapper termMapper;
+    private final PictureMapper pictureMapper;
```

Comment:
This service is getting too big, I think. I would consider splitting it to two separate services:
- DefinitionQuestionService
- PictureQuestionService

Reply:
Sure, I agree with that.

Reply:
Question Service splitted using generic interface.